openshift_scalability/ci/scripts/cnv/scale_build_test.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+
+########################################################################################################################################################
+#  This is script to run the scale testing  using cluster loader on OCP.                                                                               #
+#  It gets VMs quantity and test vm as parameters.                                                                                                     #
+#                                                                                                                                                      #
+#  Usage:                                                                                                                                              #
+#   ./scale_build_test.sh ${SETUP_PBENCH} ${CONTAINERIZED} ${CLEAR_RESULTS} ${MOVE_RESULTS} ${TOOLING_INVENTORY} ${CNV_TEST_NAME} ${VMS_QUENTITY}      #
+#                                                                                                                                                      #
+########################################################################################################################################################
+
+SETUP_PBENCH=$1
+CONTAINERIZED=$2
+CLEAR_RESULTS=$3
+MOVE_RESULTS=$4
+TOOLING_INVENTORY=$5
+TEST_NAME=$6
+VMS_QUENTITY=$7
+CURRENT_TEST=""
+CURRENT_TEST_NS=""
+
+function get_ns_names {
+  # returns array with name spaces name according to test name
+  base_name="cvn-vm-testing-"$1
+  for index in {0..20}
+  do
+    args+=$base_name"${index}"
+    args+=" "
+  done
+  echo "${args[@]}"
+}
+
+if [[ "${TEST_NAME}" == "cirros_test" ]]; then
+    CURRENT_TEST='cnv_vm_cirros'
+    declare -a CURRENT_TEST_NS=$(get_ns_names "cirros")
+
+elif [[ "${TEST_NAME}" == "fedora_test" ]]; then
+    CURRENT_TEST='cnv_vm_fedora'
+    declare -a CURRENT_TEST_NS=$(get_ns_names "fedora")
+else
+   echo "${TEST_NAME} is not a valid option, available options: cirros_test, fedora_test"
+   exit 1
+fi
+echo $CURRENT_TEST_NS
+
+
+echo " =========================================================================="
+echo "     CNV test                                                              "
+echo "     Test name: '${CURRENT_TEST}'                                          "
+echo "     VM namespace: '${CURRENT_TEST_NS}'                                    "
+echo "     Test # of VMs: '${VMS_QUENTITY}'                                      "
+echo " =========================================================================="
+
+
+## Setup pbench
```

Comment:
We are handling the tooling setup separately outside the benchmarks, so line 55-83 is not needed.